devel/sift/Portfile

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem 		1.0
+
+# PortGroup		github 1.0
+# github.setup	svent sift 0.9.0 v
+
+name            sift
+version         0.9.0
+platforms       darwin
+categories      sysutils
+license         GPL-3+
+installs_libs   no
+maintainers     esafak
```

Comment:
GItHub usernames should be prefixed with an @ character. See "maintainer" at https://guide.macports.org/#reference.keywords